projects/core/src/routing/services/url-matcher.service.ts

```diff
@@ -7,19 +7,15 @@ import {
   UrlSegmentGroup,
 } from '@angular/router';
 import { GlobService } from '../../util/glob.service';
-import { UrlMatcherFactory } from '../configurable-routes/routes-config';
 
 @Injectable({ providedIn: 'root' })
-export class UrlMatcherFactoryService {
-  constructor(
-    protected globService: GlobService,
-    protected injector: Injector // spike todo breaking change (can be avoided if we add it early)
-  ) {}
+export class UrlMatcherService {
+  constructor(protected globService: GlobService) {}
 
   /**
    * Returns a matcher that is always fails
    */
-  getFalsyUrlMatcher(): UrlMatcher {
+  getFalsy(): UrlMatcher {
```

Comment:
`getFalsy: UrlMatcher = () => null;`

Reply:
It could be:
```typescript
getFalsy(): UrlMatcher {
  return () => null
}
```

But returning a named function was on purpose - for easier debugging of routes.
![image](https://user-images.githubusercontent.com/4001059/75343206-3d9a0a00-5898-11ea-89cc-1d42e513fe5d.png)